src/components/subscriptions/LicenseActions.jsx

```diff
@@ -1,17 +1,38 @@
-import React, { useMemo } from 'react';
+import React, { useContext, useMemo } from 'react';
 import PropTypes from 'prop-types';
 
 import './styles/LicenseActions.scss';
 
+import LicenseRevokeModal from '../../containers/LicenseRevokeModal';
+import ActionButtonWithModal from '../ActionButtonWithModal';
+
+import { StatusContext } from './SubscriptionManagementPage';
+
 export default function LicenseAction({ user }) {
+  const setStatus = useContext(StatusContext);
+
+  const setRevokeStatus = () => {
+    setStatus({
+      visible: true,
+      alertType: 'success',
+      message: 'Successfully revoked license',
+    });
+  };
+
   const licenseActions = useMemo(
     () => {
       switch (user.licenseStatus) {
         case 'active':
           return [{
             text: 'Revoke',
             // eslint-disable-next-line no-console
-            handleClick: () => { console.log('Revoke clicked for', user); },
+            handleClick: closeModal => (<LicenseRevokeModal
+              user={user}
+              onSuccess={() => setRevokeStatus()}
```

Comment:
I think we can make this generic function as in remind and revoke case `onSuccess` only message is changing. So message can be passed from here.